docs/overview/migrating-react-native.md

```diff
@@ -5,35 +5,105 @@ sidebar_label: Migrating react-native
 original_id: migrating_react_native
 ---
 
-
```

Comment:
was there a reason for these spaces?

Reply:
Old text formatting. I see that it isn't needed anymore.